package/CHANGELOG

```diff
@@ -23,6 +23,8 @@ Fixes
 Enhancements
 
 Changes
+  * Reverts PR #4108, builds are now again made using the oldest
```

Comment:
I would just cut a bugfix release out of the current state of 2.7.0, rather than cherry pick - everything here seems like sensible bug fixes imho.